src/content/docs/iast/use-iast.mdx

```diff
@@ -0,0 +1,189 @@
+---
+title: New Relic interactive applications security testing (IAST)
+tags:
+  - IAST
+  - Exploitable vulnerabilities
+metaDescription: Use Vulnerability Management to find exploitable vulnerabilities in your application.
+---
+
+import iastSummary from 'images/iast_screenshot-crop_summary.webp'
+
+import iastInstall from 'images/IAST_screenshot-crop_install.webp'
+
+import iastAppVuln from 'images/IAST_screenshot-crop_app-vuln.webp'
+
+import iastAppCoverage from 'images/IAST_screenshot-crop_app-coverage.webp'
+
+import iastVulnDetails from 'images/iast_screenshot-crop_vuln-details.webp'
+
+import iastExploitablesAll from 'images/iast_screenshot-crop_exploitables-all.webp'
+
+import iastCoverageOverview from 'images/iast_screenshot-crop_coverage-overview.webp'
+
+import iastUntestedApps from 'images/iast_screenshot-crop_untested-apps.webp'
+
+import iastEnableIast from 'images/iast_screenshot-crop_enable-iast.webp' 
+
+<Callout title="PREVIEW"> 
+    This feature is currently in preview.
+</Callout>
+
+When your application has exploitable vulnerabilities, it means that someone could take advantage of a misconfiguration to access sensitive information. To help prevent that from happening, install our interactive applications security testing (IAST) to assist in finding exploitable vulnerabilities. 
```

Comment:
To help prevent that from happening, install our interactive applications security testing (IAST) to assist in finding exploitable vulnerabilities. -->

To help prevent that, install our interactive applications security testing (IAST).